### PR TITLE
Vulnerability: Multiple Atlassian Products

### PR DESCRIPTION
**Pull request by defau1t**
### For Vulnerability Reports
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by the Bug Alert team.
- [x] Ensure you [used the notice template](https://github.com/sullivanmatt/bugalert/blob/main/content/notices/202X-MM-DD-slug.md.template) for posting a notice. The 'summary' field will be used for notifications (email, phone, and SMS). It is of the utmost importance that the summary be clear and concise, so we ask that you please follow the format suggested in the template unless there is a compelling and justifiable need to deviate from it.

Leave the following intact, which notifies our volunteer team: @BugAlertDotOrg/volunteer-team

### CODE REVIEW

File: content/notices/2022-07-20-atlassian.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+Title: Servlet Filter Bypass Vulnerabilities in Multiple Atlassian Products (CVE-2022-26136, CVE-2022-26137)
+Date: 2022-07-20 10:00
+Category: Services & System Applications
+Tags: Confluence, Jira, Bitbucket, Bamboo, Fisheye, Crowd, Atlassian, Very High Severity
+Slug: atlassian
+Summary: Servlet Filter Bypass Vulnerabilities have been found in multiple Atlassian products that may enable Authentication Bypasses, XSS Vulnerabilities, and CORS vulnerabilities. These vulnerabilites have been assigned a bug alert severity of 'very high'. Atlassian recommends patching affected installations immediately.
+---
+
+On Wednesday, July 20th, 2022, [Atlassian released a Security Advisory](https://confluence.atlassian.com/doc/confluence-security-advisory-2022-06-02-1130377146.html) stating that Servlet Filter bypasses exist in both first and third party applications tied to multiple Atlassian products. At this time they have not released any specifics as to what the exact vulnerable endpoint is, or any indicators of compromise that could lead defenders to believe they have been exploited. The current fix is to patch to the level indicated in the advisory.
```
Comment (rgmz) [inline content/notices/2022-07-20-atlassian.md]: > On Wednesday, July 20th, 2022, [Atlassian released a Security Advisory](https://confluence.atlassian.com/doc/confluence-security-advisory-2022-06-02-1130377146.html)

Is this the right link? The latest update in this one seems to have been on June 10th, and there's no reference to CVE-2022-26136 or CVE-2022-26137.

Comment (defau1t) [reply content/notices/2022-07-20-atlassian.md]: Nope you're right, my vim was being helpful and putting cut text into system clipboard. I'll fix that when I push a clearer notice.

**Comment by rgmz** [comment]
I think we want: https://confluence.atlassian.com/security/july-2022-atlassian-security-advisories-overview-1142446703.html

**Comment by defau1t** [comment]
> I think we want: https://confluence.atlassian.com/security/july-2022-atlassian-security-advisories-overview-1142446703.html

We sure do! That'll be in the next push

**Review by basisbit — APPROVED** [review]
Lgtm, thanks for the report!

**Review by sullivanmatt — APPROVED** [review]

**Pull request merged**